pgjdbc/src/main/java/org/postgresql/jdbc/PgPreparedStatement.java

```diff
@@ -1445,7 +1461,11 @@ public void setTimestamp(@Positive int i, @Nullable Timestamp t,
     checkClosed();
 
     if (t == null) {
-      setNull(i, Types.TIMESTAMP);
+      if (connection.isSqlTimestamptzAlways()) {
```

Comment:
Seems we now have timestamp with and without time zone. Why force it ?

Reply:
to keep align with logic enforced by isSqlTimestamptzAlways

Reply:
AFAICT, this is the only logic with isSqlTimestamptzAlways

Reply:
it is used also below to set the oid also in case of not null. 
Line 1522:

`
if (connection.isSqlTimestamptzAlways()) {
      oid = Oid.TIMESTAMPTZ;
    }
`